packages/style-engine/src/index.ts

```diff
@@ -17,10 +17,10 @@ import { styleDefinitions } from './styles';
 /**
  * Generates a stylesheet for a given style object and selector.
  *
- * @param  style   Style object, for example, the value of a block's attributes.style object or the top level styles in theme.json
- * @param  options Options object with settings to adjust how the styles are generated.
+ * @param {Style}        style   Style object, for example, the value of a block's attributes.style object or the top level styles in theme.json
```

Comment:
Out of curiosity, is the type for `@param` enforced in TypeScript files, too? I had an impression this was lifted as we can infer the type from TS syntax.

Reply:
You're right: there's a bit of tautology here. I was think about folks who were reading the code.

Actually I see the docs are doubling up, so it's probably safe to revert these. Thanks!